terraform/modules/rds/main.tf

```diff
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
+  engine           = "postgres"
+  engine_version   = "13.7"
+  instance_class   = var.instance_class
+  allocated_storage = 20
+
+  # Non-compliant: Credentials in plain text
+  name     = var.db_name
+  username = "admin"  # Hardcoded credentials
+  password = "insecure123!"  # Hardcoded password
```

Comment:
Enable Multi-AZ deployment for the RDS instance to ensure high availability and compliance with SOC2 standards. Set `multi_az = true`.